llvm/test/tools/llvm-mca/X86/Generic/resources-avx512er.s

```diff
@@ -1,373 +0,0 @@
-# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
```

Comment:
Why removing this?

Reply:
scheduleinfo was removed. Now I think we should keep schedule info for these instructions, will add back.